### PR TITLE
Release: @slack/socket-mode@2.0.3, @slack/oauth@3.0.2, @slack/rtm-api@7.0.2

### PR DESCRIPTION
**Pull request by hello-ashleyintech**
### Summary
Releases  @slack/socket-mode@2.0.3, @slack/oauth@3.0.2, @slack/rtm-api@7.0.2 to patch security vulns

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: packages/socket-mode/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slack/socket-mode",
-  "version": "2.0.3-rc.1",
```
Comment (zimeg) [inline packages/socket-mode/package.json:3]: I messed [this merge](https://github.com/slackapi/node-slack-sdk/pull/2099/files#diff-6ce92bbc9c794026b6793cbac24c6b968f1e8e11add1bd848675f3eaad0e3702L3) up in #2099!

The current release is `2.0.2` so I think removing the "release candidate" part is alright to bump this to `2.0.3` 🏷️

**Review by zimeg — COMMENTED** [review]
Quick note on the `@slack/socket-mode` versioning with a recent change 😳

File: packages/socket-mode/package.json (hunk 1 above)
Comment (hello-ashleyintech) [reply packages/socket-mode/package.json:3]: ahhh ok gotcha!! will update 😄

**Review by zimeg — APPROVED** [review]
🙏 ✨

**Comment by hello-ashleyintech** [comment]
since it's near EOD for me today, will merge this in and release tomorrow and continue towards a new Bolt JS release ✨

**Pull request merged**